lib/castle/secure_mode.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Castle
+  module SecureMode
+    def self.signature(user_id)
+      OpenSSL::HMAC.hexdigest('sha256', Castle.config.api_secret, user_id)
```

Comment:
I think this will crash if `user_id` is an Integer. `user_id.to_s` would fix it.